src/components/Blockly/blocks/fluoroASM.js

```diff
@@ -0,0 +1,253 @@
+// path: src/components/Blockly/blocks/fluoroASM.jsx
+// Erstellt
+import * as Blockly from "blockly";
+import "blockly/javascript"; // Stellt sicher, dass Blockly.JavaScript existiert
+import { getColour } from "../helpers/colour";
+import * as Types from "../helpers/types";
+import { FieldGridDropdown } from "@blockly/field-grid-dropdown";
+import { FieldSlider } from "@blockly/field-slider";
+
+/**
+ * Fluoro-ASM Sensor und LED-Steuerung
+ */
+
+// Initialisierung des Sensors
+Blockly.Blocks["sensebox_fluoroASM_init"] = {
+  init: function () {
+    this.setColour(getColour().sensebox);
+    this.appendDummyInput().appendField("Fluoro-ASM Sensor initialisieren");
+    this.setPreviousStatement(true, null);
+    this.setNextStatement(true, null);
+    this.setTooltip("Initialisiert den Fluoro-ASM Sensor.");
```

Comment:
@georgi-s please do not hardcode text in german. Use the internationalization keys